### DISCUSSION TITLE
Failed to connect to remote extension host server

### DISCUSSION THREAD

**Discussion by deepend-dev** [community/community#3640] [Codespaces] [answered]
Suddenly getting below message when trying to connect to github codespace from VS code desktop app from any device. Works with web app.

![image](https://user-images.githubusercontent.com/55054090/112824224-e8868400-90a7-11eb-9030-5dc1ae9c518d.png)

**Comment by PranamBhat** [comment]
Hi @deepend-dev 

Try these:

What extensions do you have installed? Can you try disabling them? And do you have the "HTML CSS Support" extension? Had an issue reported with it.


1. On your remote machine, add the following line to your .bashrc file: export VSCODE_DISABLE_PROC_READING=true
2. Install [VS Code Insiders](https://code.visualstudio.com/insiders/)
3. Try to repro your original issue with VS Code Insiders.
4. Setting that environment variable will cause VS Code (insiders only right now, I'll make a proper fix that doesn't involve an environment variable for the next stable release), to not do the proc filesystem reading that we do to automatically forward ports for you. This does mean that automatic port forwarding will not work while you have that environment variable set.
5.  If still not resolved, try executing "wsl --shutdown"
6. At last, try uninstalling and then re-installing all extensions.

**Reply by deepend-dev** [reply]
@PranamBhat tried it today with no luck in connecting to github codespaces. 

- added variable and installed vscode insiders.
  
![image](https://user-images.githubusercontent.com/55054090/113160675-970bff80-9235-11eb-9f5d-ce16ef6aa21a.png)

- same issue on vscode insider

![image](https://user-images.githubusercontent.com/55054090/113160731-a68b4880-9235-11eb-9dea-98b29195a0ad.png)

![image](https://user-images.githubusercontent.com/55054090/113160827-bc990900-9235-11eb-9035-a30a4caaf071.png)

- I am not running wsl so point 5 doesnt apply

**Reply by PranamBhat** [reply]
Something fishy here! Not sure why!!

**Comment by deepend-dev** [comment, marked as answer]
I had to delete old codespace and create new one to make it work.. but it is resolved now

**Comment by PranamBhat** [comment]
@deepend-dev  Great! Kindly mark this as "**Mark as Answer**" and leave the discussion.
